### PR TITLE
finishing ai conversation feature

### LINKED ISSUE #16: Feature : AI Conversation
- [ ] Figure out calling the API
- [ ] Set up feature trigger ( textbox/button )
- [ ] Process and send data to API
- [ ] Figure out how to manage the returned data
- [ ] Process and display data to user
- [ ] Write unit tests for API responses
- [ ] Write unit tests for client side display 
- [ ] Merge with staging branch and confirm that feature works locally
- [ ] Commit and push branch 
- [ ] Create PR to merge into staging

### PR DESCRIPTION
**Pull request by franollf**
finished the ai conversation feature

### CODE REVIEW

**Comment by gunmack** [comment]
Might want to make a component in `pages/api` to handle Gemini API interaction. Right now, the API key would be exposed to the browser due to having the `NEXT` keyword. Essentially just do the same thing you did for Text-to-speech

File: src/app/ai-conversations/page.js
Comment (gunmack) [inline src/app/ai-conversations/page.js:1]: Make a file in `pages/api` to handle API interaction
`NEXT_PUBLIC` would expose API key on the browser

**Review by gunmack — CHANGES_REQUESTED** [review]

**Comment by gunmack** [comment]
Also I'm wondering why the workflow didn't run for this pr, did you fork the repo or clone it?

**Comment by franollf** [comment]
no idea will fix it later

**Review by gunmack — DISMISSED** [review]

**Review by garrett-darts — DISMISSED** [review]

**Review by garrett-darts — APPROVED** [review]

**Pull request merged**